components/ai_chat/core/browser/engine/engine_consumer_oai.cc

```diff
@@ -88,6 +88,9 @@ base::Value::List BuildMessages(
     message.Set("role", turn->character_type == CharacterType::HUMAN
                             ? "user"
                             : "assistant");
+    const std::string& text = (turn->edits && !turn->edits->empty())
```

Comment:
In other places we don't test `turn->edits`, we just check whether its empty? Do we need to update the other places to do this check too?

In the mojom, it seems to be nullable, so maybe we should? Alternatively, we could just make it a required field (and set it to the empty array)

Reply:
Sorry would you mind point me where we only check it's empty in c++?

Reply:
Sorry I think I must've hallucinated this - I had a look but can't seem to find it! Sorry!